Doc/library/unittest.rst

```diff
@@ -220,6 +220,25 @@ Command-line options
 
    See `Signal Handling`_ for the functions that provide this functionality.
 
+.. cmdoption:: --debug
+
+   On test fail or error the test run terminates immediately with original
+   exception - similar to normal script execution. Useful for seamless external
+   post-mortem handling.
+
+.. cmdoption:: --pdb
+
+   Runs ``pdb.post_mortem()`` upon each error. Short for ``--pm=pdb``
```

Comment:
Could this be a link to pdb.post_mortem doc?

Reply:
Made it ``` :meth:`pdb.post_mortem` ``` for next push